app/models/brand.rb

```diff
@@ -28,6 +28,6 @@ def from_param(param)
   end
 
   def to_param
-    "#{name.parameterize}"
+    name.parameterize.to_s
```

Comment:
```suggestion
    name.parameterize
```

Reply:
i think it's already a string coming from parameterize

Reply:
Done.